tests/test-war-smoke/src/test/java/com/google/cloud/runtimes/jetty/tests/JspRemoteIntegrationTest.java

```diff
@@ -0,0 +1,25 @@
+package com.google.cloud.runtimes.jetty.tests;
+
+
+import com.google.cloud.runtime.jetty.testing.AppDeployment;
+import com.google.cloud.runtime.jetty.testing.HttpUrlUtil;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class JspRemoteIntegrationTest extends AbstractJspIntegrationTest {
+
+  @BeforeClass
+  public static void isServerUp() {
+    HttpUrlUtil.waitForServerUp(AppDeployment.SERVER_URI, 5, TimeUnit.MINUTES);
```

Comment:
Testing for server up seems to be a common theme across all of these tests. Maybe we can pull this up into a parent class like `AbstractIntegrationTest`. Also, AppDeployment.SERVER_URI should probably be made to default to localhost, so that we can use a single `isServerUp()` for both local and remote tests.